docs/content/how-to/kubevirt/external-infrastructure.md

```diff
@@ -113,4 +115,12 @@ rules:
       - secrets
     verbs:
       - '*'
+  - apiGroups:
+    - snapshot.storage.k8s.io
+    resources:
+    - volumesnapshots
+    verbs:
+    - get
+    - create
+    - delete
```

Comment:
the infra role  in this PR includes more than snapshots
```
- apiGroups: ["snapshot.storage.k8s.io"]
  resources: ["volumesnapshots"]
  verbs: ["get", "create", "delete"]
- apiGroups: [""]
  resources: ["persistentvolumeclaims"]
  verbs: ["get"]
```

which is right? do we need persistentVolumeClaims as well? if so, this doc should be updated

Reply:
yes we need pvcs as well.

Reply:
https://github.com/openshift/kubevirt-csi-driver/blob/master/pkg/kubevirt/client.go#L367